src/routes/+layout.svelte

```
@@ -1,7 +1,38 @@
+	export const load = async () => {
+		if (browser) {
+			console.log('Loading PostHog...');
+			posthog.init('phc_KSsuPOYsQfYrMaMH0nXb1PSMkgYS0cTKU2hv7dKmGmn', {
+				api_host: 'https://eu.i.posthog.com',
+				defaults: '2025-05-24',
+				person_profiles: 'always'
+			});
+		}
+
+		return;
+	};
```

ESLint

[error] disallow props other than data or errors in SvelteKit page components.

(svelte/valid-prop-names-in-kit-pages)